packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -33,9 +33,11 @@
         <action android:name="android.intent.action.VIEW" />
         <category android:name="android.intent.category.DEFAULT" />
         <category android:name="android.intent.category.BROWSABLE" />
-        <data android:scheme="https" android:host="valoraapp.com" android:pathPrefix="/wc" />
+        <data android:scheme="https" android:host="valoraapp.com" />
+        <data android:pathPrefix="/wc" />
```

Comment:
can you do some basic manual test to make sure this doesn't break WC on Android?